pygfx/renderers/wgpu/shaders/meshshader.py

```diff
@@ -143,7 +150,9 @@ def get_bindings(self, wobject, shared):
             bindings.append(Binding("s_colors", rbuffer, geometry.colors, "VERTEX"))
         elif self["color_mode"] in ("vertex_map", "face_map"):
             bindings.extend(
-                self.define_texcoords_and_colormap(
+                # todo: check uv is present and has the right shape. (introduce uv channel for texture)
+                # todo: unify the logic with other maps, introduce uv channel and texture sampler
```

Comment:
Is this todo still relevant? Looks like the texcoords are handled correctly, and it is checked for compatibility with the map in `define_colormap()`.

Reply:
My idea is to introduce the `uv_channel` attribute in textures first (to identify different texcoord sets), and based on `uv_channel`, determine if the corresponding texcoord exists and matches the shape of this texture.

Once these steps are completed, the logic for the `colormap` should align with that of other textures. This is just a record of the plan.